test/e2e/containers_conf_test.go

```diff
@@ -252,7 +252,7 @@ var _ = Describe("Podman run", func() {
 		session := podmanTest.Podman([]string{"run", ALPINE, "cat", "/etc/resolv.conf"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
-		Expect(session.LineInOutputStartsWith("search")).To(BeTrue())
+		Expect(session.OutputToString()).To(HavePrefix("search"))
```

Comment:
This is not correct, `LineInOutputStartsWith` checks each line individually for this prefix while your change only works for the first line.

Reply:
yes, exactly. I'm looking for a Ginkgo replacement.

Reply:
I think you need to write your own custom matcher